src/main/java/tictactoe/PlayerTypes.java

```diff
@@ -0,0 +1,12 @@
+package tictactoe;
+
+public enum PlayerTypes {
```

Comment:
Do you need anything more than:

```java
public enum PlayerType {
  COMPUTER,
  HUMAN;
}
```

Reply:
Currently the user enters `"h"`, or `"c"` when the app begins to specify the type of players for the game. However, I can change this, so the user needs to type `human` or `computer`